internal/admission/validation/gateway/httproute.go

```diff
@@ -289,3 +289,30 @@ func validateWithKongGateway(
 func validationMsg(errMsgs []string) string {
 	return fmt.Sprintf("HTTPRoute failed schema validation: %s", strings.Join(errMsgs, ", "))
 }
+
+func validateHTTPRouteTimeoutBackendRequest(httproute *gatewayapi.HTTPRoute) error {
+	// TODO: remove the validate after we figure out how to handle granular timeout settings
+	// (allowing setting timeouts per rule and not enforcing the same timeout for every HTTPRoute's rule).
+	// https://github.com/Kong/kubernetes-ingress-controller/issues/5451
+
+	timeout := lo.ToPtr(gatewayapi.Duration("60s"))
+	isTimeoutSet := false
+
+	for _, rule := range httproute.Spec.Rules {
+		if isTimeoutSet {
+			if rule.Timeouts == nil {
+				return fmt.Errorf("timeout is set for one of the rules, but not set for another")
+			}
+			if rule.Timeouts != nil && *rule.Timeouts.BackendRequest != *timeout {
+				return fmt.Errorf("timeout is set for one of the rules, but a different value is set in another rule")
+			}
+		} else {
+			if rule.Timeouts != nil && rule.Timeouts.BackendRequest != nil {
+				timeout = rule.Timeouts.BackendRequest
+				isTimeoutSet = true
+			}
+		}
+	}
+
+	return nil
```

Comment:
I think we could avoid using a separate boolean if we do not initialize the timeout variable with `60s`.
```suggestion
	var firstTimeoutFound *gatewayapi.Duration
	for _, rule := range httproute.Spec.Rules {
		if firstTimeoutFound != nil {
			if rule.Timeouts == nil {
				return fmt.Errorf("timeout is set for one of the rules, but not set for another")
			}
			if rule.Timeouts != nil && *rule.Timeouts.BackendRequest != *firstTimeoutFound {
				return fmt.Errorf("timeout is set for one of the rules, but a different value is set in another rule")
			}
		} else {
			if rule.Timeouts != nil && rule.Timeouts.BackendRequest != nil {
				firstTimeoutFound = rule.Timeouts.BackendRequest
			}
		}
	}

	return nil
```